### PR TITLE
[WIP] Azure Setup docs for using Docker 

### PR DESCRIPTION
**Pull request by lotusgraham**

### CODE REVIEW

**Comment by ckcollab** [comment]
Think we can merge this, 👍

File: docs/1. Setup Guide - Docker.md [hunk 1]
```diff
@@ -190,8 +190,8 @@ You may sign up for an Azure account [here](https://azure.microsoft.com/en-us/),
 
 1. You do not have to do this if you've already setup S3!
 1. Log on to the [Azure Portal](https://portal.azure.com).
-1. In the left pane, click **Storage**.
-1. Select your storage account.
+1. From the Dashboard, click **Storage accounts** on the left.
+1. Click Add at the top of the page to create a new storage account.
```
Comment (lotusgraham) [inline docs/1. Setup Guide - Docker.md]: Make both blob containers, and name one public and one private.
image

Comment (lotusgraham) [inline docs/1. Setup Guide - Docker.md]: image

File: docs/1. Setup Guide - Docker.md [hunk 2]
```diff
@@ -190,8 +190,8 @@ You may sign up for an Azure account [here](https://azure.microsoft.com/en-us/),
 
 1. You do not have to do this if you've already setup S3!
 1. Log on to the [Azure Portal](https://portal.azure.com).
-1. In the left pane, click **Storage**.
-1. Select your storage account.
+1. From the Dashboard, click **Storage accounts** on the left.
+1. Click Add at the top of the page to create a new storage account. 
 1. At the bottom of the dashboard, click **Manage Access Keys**. Copy your account name and access key to `.env` under `AZURE_ACCOUNT_NAME` and `AZURE_ACCOUNT_KEY`.
 1. At the top of the dashboard page, click **Containers**.
 1. At the bottom of the Containers page click **Add**.
```
Comment (lotusgraham) [inline docs/1. Setup Guide - Docker.md]: image

**Comment by ckcollab** [comment]
👍

**Pull request merged**